pkg/ipam/crd.go

```diff
@@ -170,7 +174,7 @@ func newNodeStore(nodeName string, owner Owner, k8sEventReg K8sEventRegister) *n
 		time.Sleep(5 * time.Second)
 	}
 
-	store.refreshTrigger.TriggerWithReason("initial sync")
+	go store.refreshTrigger.WaitAndTriggerWithReason("initial sync", store.restoreFinished)
```

Comment:
This was very hard to read. I guess you changed it to this based on my feedback, apologies for that.

I think this would be cleaner if we write:
```
go func() {
     <- store.restoreFinished
     store.refreshTrigger.TriggerWithReason("initial sync")
}()